examples/split_examples/spynnaker.cfg

```diff
@@ -0,0 +1,2 @@
+[Mode]
+violate_1ms_wall_clock_restriction = True
```

Comment:
Is this violate parameter necessary for every example it's been added for (both here and in IntroLab)?

Reply:
It is needed where time_scale_factor is set to 1 and timestep is set to 0.1.  Since there is one example in this folder which does this, I just set it here.  It could be split into two different folders, but setting it doesn't break anything where it isn't needed (this just allows the violation).  In IntroLabs, the learning examples probably don't need it, but equally it doesn't really matter; it might be desired in the future if we decide to run those at 0.1ms too I guess.